code/game/objects/items/toys.dm

```diff
@@ -343,7 +341,6 @@
 	flags_1 = CONDUCT_1
 	slot_flags = SLOT_BELT | SLOT_BACK
 	force = 5
-	throwforce = 5
```

Comment:
undocumented change

Reply:
reverted and documented for flavor text.